utils/regex.js

```diff
@@ -4,7 +4,7 @@ const niNumber  = /^\s*[a-zA-Z]{1}\s*[a-zA-Z]{1}\s*(?:\s*\d\s*){6}[a-zA-Z]{1}\s*
 const title = /^[a-zA-Z /&]{2,}$/;
 const firstName = /^[a-zA-z]+([-][a-zA-Z]+)*$/;
 const lastName  = /^[a-zA-z]+([ '-][a-zA-Z]+)*$/;
-const whitelist = /^[a-zA-Z0-9 .",'?!()£:-]{2,}$/;
+const whitelist = /^[a-zA-Z0-9 \r\n.",'?!()£:-]{2,}$/;
```

Comment:
Just a Question for everyone (@iguannago @paulgain ) Should we maybe create a new regex similar to this for text areas, as we probably don't want to allow new lines in text fields?

Reply:
that's a good point